build-system/compile/compile-wrappers.js

```diff
@@ -74,7 +66,6 @@ function extensionPayload(name, version, latest, isModule, loadPriority) {
     `v:"${VERSION}",` +
     `n:"${name}",` +
     `ev:"${version}",` +
-    `l:${latest},` +
```

Comment:
See also code in `src/runtime.js` and `src/service/extensions-impl.js`

Reply:
Fixing this is a bit hairy/scary in an untyped world

Reply:
Don't plan on merging until @rsimha is back anyway, so I guess I may as well take a stab at it

Reply:
I'm actually unsure of what our desired behavior here is re. reloading extensions etc...going to revert back to before any runtime changes, can clean up and delete code after the build-side cleanup is done